### ISSUE TITLE
[validator/validator.d.ts] Type expected.

### ISSUE THREAD

**Issue by michaeljota** [DefinitelyTyped/DefinitelyTyped#11510]
- [ ✔️ ] I tried using the latest `validator/validator.d.ts` file in this repo and had problems.
- [ 😞 ] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [ ✔️ ] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [ ✔️ ] I want to talk about `validator/validator.d.ts`.
  - The authors of that type definition are cc/ @tgfjt @chrootsu @IOAyman @louy

Lastest update give me type errors. (line 186 ~ 187)

```typescript
    // canonicalize an email address.
    normalizeEmail(email: string, options?: NormalizeEmailOptions): string | false;
``` 

I guess it is hopping a type, not a literal boolean value. So I guess it should be: `string | boolean`. (That was my workaround at least)

**Comment by andy-ms** [comment]
Sounds like you're using the npm package `@types/validator`. (The `validator.d.ts` on the master branch does not have `| false`, so I infer that you are using the `types-2.0` version, which is published as `@types/validator`.)
`@types` packages are only compatible with TypeScript 2.0, which adds support for boolean literal types. More info [here](https://blogs.msdn.microsoft.com/typescript/2016/09/22/announcing-typescript-2-0/).

**Comment by michaeljota** [comment]
You are right, I'm using `@types/validator`, but I'm also using TypeScript 2.0. I know it support string literal as types, but I don't know why it complains about boolean literals. However, it does not compile the project.

**Comment by andy-ms** [comment]
It looks like this wasn't available in `2.0.0` but is in `2.0.2` and above.

**Comment by michaeljota** [comment]
Yup. Updated to lastes (`2.0.3`) and now it's working. I'm sorry.
